epoch/api/contract_api_usage.md

```diff
@@ -0,0 +1,45 @@
+[back](./README.md)
+# Contracts - intended usage
+
+## Introduction
+You interact with a Aeternity node through HTTP.
+To learn more about contracts and contract life cycles see [the doc](/contracts/contracts.md).
+
+The following assumes that the node exposes at address 127.0.0.1 the following ports:
+* HTTP external API: 3013
+* HTTP internal API: 3113
+
+There are two basic types of API calls, off-chain operations and on-chain transactions.
+
+In order to affect the state of the chain you have to submit a signed transaction
+to a mining node.
+
+For an up to date list of all HTTP API endpoints and their arguments see
+[Epoch HTTP API](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/master/apps/aehttp/priv/swagger.json)
+
+## Off chain operations
+
+An epoch node provides some utility functions to help you create contract transactions.
+
+
+### Contract Compile 
+[/contract/compile doc](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/master/apps/aehttp/priv/swagger.json#/external/CompileContract)
```

Comment:
I am a bit concerned re linking master from this repo, as this would not play nicely with tagging this repo. Though as a draft may be ok.

Reply:
Or some other way to refer to the generated doc?

Reply:
Will polish afterwards. Thanks.